pa_bin/performance-analyzer-agent

```diff
@@ -15,14 +15,23 @@ else
 fi
 
 if [ -z "$2" ]; then
-  if [ -z "$JAVA_HOME" ]; then
-    echo "JAVA_HOME needs to be set or passed in as the second parameter."
-    exit 1
+  # Check if OPENSEARCH_JAVA_HOME is present
+  if [ -z "$OPENSEARCH_JAVA_HOME" ]; then
+      # If OPENSEARCH_JAVA_HOME is not present, check JAVA_HOME next
+      if [ -z "$JAVA_HOME" ]; then
+        echo "OPENSEARCH_JAVA_HOME / JAVA_HOME needs to be set or passed in as the second parameter."
+        exit 1
+      fi
+  else
+    JAVA_HOME=$OPENSEARCH_JAVA_HOME
   fi
 else
   JAVA_HOME=$2
 fi
```

Comment:
These nested ifs are getting very confusing. Maybe swap them?

1. If $2 use that
2. Else if OPENSEARCH_JAVA_HOME use that
3. Else if JAVA_HOME use that
4. Else fail?

Reply:
:+1: done!